### PR TITLE
Add permanent sites table

### PR DESCRIPTION
**Pull request by KatalinaBobowik**
Add sites table from `cpg-common-main`, instead of pulling the sites table from `hgdp-1kg-test` or `hgdp-1kg-main`. 

For context, see the [PR](https://github.com/populationgenomics/references/pull/42#pullrequestreview-1810428108) to get the sites table onto `cpg-common`, as well as [this](https://centrepopgen.slack.com/archives/C030X7WGFCL/p1704684891602139) slack thread for context.

### CODE REVIEW

**Comment by MattWellie** [comment]
You could cut this from the workflow-specific config completely now that it's in the default config file, and grab it through
`get_config()['references']['ancestry']['sites_table']`

**Review by vivbak — CHANGES_REQUESTED** [review]
Second Matt's comment above. Could we also check that we've pulled the latest, r.e. failing tests

**Comment by KatalinaBobowik** [comment]
Thanks @MattWellie , I've added that in!

@vivbak the tests are now fixed 👌

File: configs/defaults/large_cohort.toml [hunk 1]
```diff
@@ -75,6 +75,4 @@ delete_scratch_on_exit = false
 #combiner_autoscaling_policy = 'vcf-combiner-50'
 
 [references.gnomad]
-tel_and_cent_ht = "gs://cpg-common-main/references/gnomad/v0/telomeres_and_centromeres/hg38.telomeresAndMergedCentromeres.ht"
-predetermined_qc_variants_prefix = 'gs://cpg-hgdp-1kg-' 
-predetermined_qc_variants_path = '/sites_table/v1-0/pruned_variants.ht'
+tel_and_cent_ht = "gs://cpg-common-main/references/gnomad/v0/telomeres_and_centromeres/hg38.telomeresAndMergedCentromeres.ht"
```
Comment (MattWellie) [inline configs/defaults/large_cohort.toml:78]: This also present by default in the config file

**Review by vivbak — APPROVED** [review]

**Pull request merged**